component/cluster.libsonnet

```diff
@@ -156,6 +156,24 @@ local cluster = function(name, options)
     },
   };
 
+  local initManifestsCM = kube.ConfigMap(name + '-init-manifests') {
```

Comment:
Since I don't see any references to this configmap in the component, I assume this is a magic name which is picked up by vcluster?

Reply:
Yes it's a magic name that the vcluster syncer picks up on.